### ISSUE TITLE
image not found - message

### ISSUE THREAD

**Issue by goessner** [goessner/g2-new#13]
while using the `g2.img` command the error message (Firefox) reads

```
NS_ERROR_NOT_AVAILABLE: g2.js:334
```
The broken image symbol should be seen here instead.

the calling source reads 
```js
 g.img({uri:this.uri,x:()=>this.p.x,y:()=>this.p.y,w,scl:this.scl,xoff:this.xoff,yoff:this.yoff})
```

**Comment by klawr** [comment]
Is this the same problem with images taking too long to load to show them?
[If you refresh the page often enough the not found image (#28) shows up in the test-page sometimes.](https://goessner.github.io/g2-new/index.html).

However the atom.png never shows up.
Have we decided on a solution for images? Like a fixed delay of 100ms for images or similar.

**Comment by goessner** [comment]
A fixed delay of 100 ms sounds like a good idea ..

**Comment by klawr** [comment]
I think i came across a much better solution:

I delete this part:
```js
 if (img.complete)
           this.ctx.drawImage(img,0,0);
        else // broken image ..
           this.ctx.drawImage(getImg("data:image/gif;base64,R0l..."),0,0);
```

and add 2 Events to the img-Object, so it now looks like this:
```js
let img = new Image();
            img.src = uri;
            img.addEventListener('error',() => this.ctx.drawImage(getImg("data:image/gif;base64,R0l..."),0,0));
            img.addEventListener('load',() => this.ctx.drawImage(img,0,0));
```

I tested it and it works and if i am not mistaken it is more performant and works better both with and without animation rendering.

**Comment by goessner** [comment]
Sounds great. Thanks

We should start to publish g2-new as 'g2'. How should we preceed here .. what do you suggest ?

**Comment by klawr** [comment]
Since images work now in a satisfying manner I think g2-new could replace g2(-old).
I will add explanations for the usage of image to the wiki soon.
Fixed with aed22adb87ea66f1f1d73a8e6e8924023a98d6d8
